README.md

```diff
@@ -68,7 +69,7 @@ The project is based on Ubuntu 16.04, ROS Kinect, gazebo 7.0, wstool and catkin
 	```
   	roscd gilbreth_gazebo
   	source scripts/env_setup.bash
```

Comment:
I think  running this script is no longer necessary, can you test that?

Reply:
I have tested this and it seems that you are correct, this script is unnecessary to run the gazebo environment.